### DISCUSSION TITLE
Display Process:: facade status

### DISCUSSION THREAD

**Discussion by cloudwales** [livewire/livewire#6150] [1. Help] [answered]
Hi. I’m building a bit of a proof of concept app using Livewire and it involves using the Process facade and returning some sort of count/progress that updates live while the process is running. 

I can use the 
‘’’
public $message;

public function runProcess(){
    $process = Process::start(“run task”);

    while($process->running){
        $this->message = “working”;
        sleep(3);
    }

   $this->message = “complete”;

}
’’’

The issue I’ve been facing is the component isn’t reloading or displaying the messages. Even if I add a message at the top of the function it won’t display till the code has finished running. Any ideas on how to show a progress of a process in the component?

cheers

**Comment by PhiloNL** [comment, marked as answer]
You might want to look at streaming the output https://youtu.be/U-N8Qqq02b0?si=moq0qtUdRKMuGLLw&t=2127

**Reply by cloudwales** [reply]
Ah that looks exactly what’s needed. 
many thanks
